openapi/index.yaml

```diff
@@ -95,12 +95,40 @@ paths:
         - application/json
       produces:
         - application/json
+    get:
```

Comment:
It seems that this endpoint is missing `x-user-id` parameter. It seems also other endpoints are missing it. Can you check? 
Also, I'd decide for a pagination mechanism and put the parameters here in the specs.